math/gfm/math/vector.d

```diff
@@ -93,6 +93,15 @@ nothrow:
             }
         }
 
+        size_t toHash() const nothrow @safe
+        {
+            size_t hash = 0;
+            foreach (elem; v) {
+                hash = elem.hashOf(hash);
+            }
+            return hash;
```

Comment:
[`hashOf`](https://dlang.org/phobos/object.html#.hashOf) is fully generic, so this should also work:

```D
return v.hashOf;
```